### ISSUE TITLE
[BUG] I can not login into my local phpmyadmin when both darkreader and keepassxc are enabled  in firefox

### ISSUE THREAD

**Issue by nonamereq** [darkreader/darkreader#4763]
### **Describe the bug**

I can't login into phpmyadmin when both dark reader and keepassxc are enabled in [about:addons](about:addons) in Firefox. The error message says failed to set session cookie.

### **Steps to reproduce**
- Only enable Dark reader and [keepassxc-browser](https://addons.mozilla.org/en-US/firefox/addon/keepassxc-browser/) addons in firefox
- Go to your phpmyadmin page and try to login

### **Expected behavior**

You login to phpmyadmin.

### **Actual behavior**

Phpmyadmin shows an error.

### **Screenshots**

[image](https://user-images.githubusercontent.com/16020706/104424014-b7051e80-5576-11eb-89d0-19797a45759c.png)

### **System Information:**

- OS: Arch Linux
- Browser: Firefox 84.0.2
- Dark Reader version: 4.9.26

### **Additional context**

I use keepassxc-browser addon with dark reader.

**Comment by Gusted** [comment]
Hi @nonamereq!

Does this work when you have Dark Reader disabled?

Regards,
Gusted

**Comment by nonamereq** [comment]
Yes it works. It doesn't work when both are enabled. When one is disabled it works.

**Comment by Gusted** [comment]
> Yes it works. It doesn't work when both are enabled. When one is disabled it works.

So if e.g. Keepassxc is enabled and Dark Reader is disabled it doesn't raise an error and visa-versa?
I'm trying to understand how this can happen as this doesn't add up for me.

**Comment by nonamereq** [comment]
> So if e.g. Keepassxc is enabled and Dark Reader is disabled it doesn't raise an error and visa-versa?

Yes that's right

**Comment by Gusted** [comment]
Hmm okay, does it show any errors in the console?

**Comment by nonamereq** [comment]
If I enable both
![2021-01-13-225316_1918x1059_scrot](https://user-images.githubusercontent.com/16020706/104503580-2875b900-55d9-11eb-950d-1c83d5c0f563.png)

If I disable dark reader
![2021-01-13-225359_1918x1059_scrot](https://user-images.githubusercontent.com/16020706/104503596-2e6b9a00-55d9-11eb-9746-f72bb02332da.png)

**Comment by nonamereq** [comment]
All I see as far as errors go is some  cookies being rejected. And that is in both cases

**Comment by nonamereq** [comment]
I found (this)[https://stackoverflow.com/questions/50968326/failed-to-set-session-cookie-maybe-you-are-using-http-instead-of-https-to-acces] question, and if i set phpmyadmin auth_type to http i can login in but not in cookie mode

**Comment by Gusted** [comment]
I'm unsure why Dark Reader is causing to Keepassxc(Due to cookies that we don't modify neither have access to and filling the input field) to let this error happen, I think you will have more success solving this issue over at https://github.com/keepassxreboot/keepassxc-browser/issues

**Comment by Optimus1008** [comment]
Got the same error, and it drove me nuts, but I finally found out that Dark Reader is causing this!
Tested it on a few different machines and different browsers, and it's safe to say that it's caused solely by Dark Reader…
The workaround is thankfully pretty easy: just disable the filter (i.e. with a shortcut), login and enable again!
I also found out that it's only caused by the dynamic style generator; the others (static, filter(+)) don't cause the issue.
I'm not nearly enough knowledgeable to dig into this any deeper, but I hope I either helped to get this fixed or helped someone like me to find a workaround. :-)

**Comment by dverbru** [comment]
@nonamereq although you found a workaround, the bug is still there, and affect other sites which we have no control over. For example, it is not possible to log in to westernunion.com. Can you please reopen it?

@Gusted since I installed Dark Reader yesterday, I found two websites I can't log into. Also on Firefox and using KeepassXC. Never had such problem before on those sites (accessed them regularly for years). I think this deserves some investigation, since it gives quite a bad taste regarding security. Thank you!

**Comment by nonamereq** [comment]
I have removed this plugin that's why i closed the issue. @dverbru I think it is better if you open a new issue

**Comment by nonamereq** [comment]
I have removed this plugin that's why i closed the issue. @dverbru I think it is better if you open a new issue

**Comment by Gusted** [comment]
> @Gusted since I installed Dark Reader yesterday, I found two websites I can't log into. Also on Firefox and using KeepassXC. Never had such problem before on those sites (accessed them regularly for years). I think this deserves some investigation, since it gives quite a bad taste regarding security. Thank you!

I'm not an KeepassXC developer, I cannot make a guess how and why dark reader is interfering with them. Nor have I a local reproducer to report this to them.
